tickets/src/components/content/EventContent.js

```diff
@@ -58,22 +58,43 @@ export class EventContent extends Component {
 
     if (!lock.address || !event.name) return null // Wait for the lock and event to load
 
-    const { name, description, location, date, links, image } = event
+    const { name, description, location, date, links = [], image } = event
     let dateString =
       MONTH_NAMES[date.getMonth()] +
       ' ' +
       date.getDate() +
       ', ' +
       date.getFullYear()
 
-    const externalLinks = (links || []).map(({ href, title }) => {
+    const externalLinks = links.map(({ href, title }) => {
       return (
         <li key={href}>
-          <a href={href}>{title}</a>
+          <a target="_blank" rel="noopener noreferrer" href={href}>
+            {title}
+          </a>
         </li>
       )
     })
 
+    const formatDateForCalendar = date => {
```

Comment:
It looks like lines 79 through 97 could be pulled into a helper function just to avoid expanding the render method too greatly. Matter of taste though.

Reply:
Good point.